tools/alpine/versions.x86_64

```diff
@@ -1,239 +0,0 @@
-# linuxkit/alpine:34af9cb1990debd17fae6d4198c62ce3910d9908
```

Comment:
Ah, missed that. Shouldn't remove it, but add an updated one once the overall Makefile changes are ok and I have updated the PR